### PR TITLE
bump version to react@17.0.0

### PR DESCRIPTION
**Pull request by Tim4497**
First Pull Request ever so please don't hate me :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "react-native-job-queue",
     "title": "React Native Job Queue",
-    "version": "0.4.1",
+    "version": "0.4.2",
```
Comment (SimonErm) [inline package.json:4]: The version gets bumped on master by the release process with `npm version` so you don't have to :-)

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "licenseFilename": "LICENSE",
     "readmeFilename": "README.md",
     "peerDependencies": {
-        "react": "^16.5.0",
```
Comment (SimonErm) [inline package.json:34]: I think we can remove the react peerDepency completly since it's unused :-)

**Comment by SimonErm** [comment]
I am glad to see :-) Your always welcome :-)

**Comment by Tim4497** [comment]
Awesome thanks

**Pull request closed without merging**